frontend/src/pages/destructionlist/abstract/BaseListView.tsx

```diff
@@ -176,7 +177,21 @@ export function BaseListView<T extends Zaak = Zaak>({
           },
         ] as ButtonProps[])
       : [];
-    return [...dynamicItems, ...fixedItems];
+    const clearFiltersItem = {
+      children: (
+        <>
+          <Solid.XCircleIcon />
+          Filters wissen
+        </>
+      ),
+      variant: "warning",
+      wrap: false,
+      onClick: () => {
+        const pageParam = searchParams.get("page") || "1";
```

Comment:
If we clear the filters, the result set may change, so page 5 (for example) wouldn't necessarily have the same meaning anymore, nor does it necessarily exist. So the 1 is not a default but a reset. I don't think we have to check if the page is currently set?